website/source/docs/providers/aws/r/cloudtrail.html.markdown

```diff
@@ -71,6 +71,8 @@ The following arguments are supported:
     Setting this to `false` will pause logging.
 * `include_global_service_events` - (Optional) Specifies whether the trail is publishing events
     from global services such as IAM to the log files. Defaults to `true`.
+* `is_multi_region_trail` - (Optional) // Specifies whether the trail is created in the current
```

Comment:
Is there any reason for those slashes here? `//` :wink: If not, please remove those.